CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 All notable changes to this project will be documented in this file.
 
+## [4.9.0] - TBD
+
```

Comment:
```suggestion

Wazuh commit: TBD \
Release report: TBD

```

Reply:
It has been decided that these references will no longer be maintained since 4.7.0.